java/code/src/com/suse/manager/webui/utils/SparkApplicationHelper.java

```diff
@@ -14,6 +14,8 @@
  */
 package com.suse.manager.webui.utils;
 
+import static com.google.gson.internal.$Gson$Types.newParameterizedTypeWithOwner;
```

Comment:
I'm not sure if it's a good idea to use methods from this package as it's obviously marked internal.

From `com.google.gson.internal` Javadoc:
> Do NOT use any class in this package as they are meant for internal use in Gson. These classes will very likely change incompatibly in future versions. You have been warned.